warehouse/manage/views.py

```diff
@@ -547,6 +549,22 @@ def manage_project_roles(project, request, _form_class=CreateRoleForm):
                     submitted_from=request.remote_addr,
                 ),
             )
+
+            owners = request.db.query(Role).filter(Role.role_name == 'Owner',
+                                                   Role.project == project)
+            email_recipients = [owner.user.email for owner in owners]
```

Comment:
We can reduce some of the memory required to do this by moving the `.user` part into the DB with a join:

```python
owners = (
    request.db.query(Role)
    .join(User)
    .filter(Role.role_name == 'Owner', Role.project == project)
)
owner_emails = [owner.email for owner in owners]
```

That way we just `User`s back from our query, not `Roles`.